lib/thrift/protocol/binary.ex

```diff
@@ -11,20 +11,21 @@ defmodule Thrift.Protocol.Binary do
   alias Thrift.TApplicationException
 
   @typedoc "Binary protocol field type identifier"
-  @type type_id :: (2..15)
+  @type type_id :: 0 | 2 | 3 | 4 | 6 | 8 | 10 | 11 | 12 | 13 | 14 | 15
```

Comment:
0/STOP is considered a field type in the Apache Thrift implementation, but I agree that it isn't useful to include in this list.